### PR TITLE
Add NextDNS.io

### PR DESCRIPTION
**Pull request by ColinHebert**
NextDNS.io, [ranked 163,067](https://www.alexa.com/siteinfo/nextdns.io) is providing an alternative DNS (similar to Cloudflare's 1.1.1.1, or Google's 8.8.8.8) with ad-blocking support as well as other security features.

Category-wise, I've put it in `Security`, similar to AdGuard (which supports DNS resolvers), and DNSFilter.

To enable, go in the account settings:
Screen Shot 2022-01-15 at 09 44 35

And use your authenticator system to scan the QRCode (QRCode and Seed are redacted in this screenshot)
Screen Shot 2022-01-15 at 09 45 02

### CODE REVIEW

File: entries/n/nextdns.io.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "NextDNS": {
+    "domain": "nextdns.io",
+    "url": "https://my.nextdns.io/",
```
Comment (ApCoder123) [inline entries/n/nextdns.io.json]:
```suggestion
```

We usually don't link to login pages on the site.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

File: img/n/nextdns.io.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg width="1024" height="1024" xmlns="http://www.w3.org/2000/svg"><circle cx="512" cy="512" r="512" style="fill:#05f"/><path d="M381 603a380 380 0 0 1-57-228 495 495 0 0 0 190-84c35 26 75 47 116 63L381 603zm285-237L401 631c30 40 67 75 110 102 65-42 119-101 153-171 28-58 40-124 36-188l-34-8zm76 34c-3 72-23 144-58 207-37 64-88 119-149 161l-9 6-6 4-6 4h-4c-50-31-92-70-128-115a463 463 0 0 1-100-315c0-5 4-9 9-10a569 569 0 0 0 220-99s3-3 6 0l19 14c56 37 118 65 184 81l17 4c5 2 5 8 5 12v46zm-19-42c-75-16-146-49-209-93a595 595 0 0 1-213 95c-5 88 19 178 67 253 37 58 86 109 144 147 74-48 135-117 172-198 29-64 42-134 38-204z" style="fill:#fff"/></svg>
```
Comment (phallobst) [inline img/n/nextdns.io.svg]: This logo does not seem to be up-to-date anymore. I haven't found an SVG version, so we might have to use a PNG icon instead.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Please use this PNG icon instead of the SVG, because as @phallobst mentioned, the SVG logo is outdated and please add the following to the JSON file: `"img": "nextdns.io.png"`.

nextdns io

File: img/n/nextdns.io.svg (hunk 2 above)
Comment (ColinHebert) [reply img/n/nextdns.io.svg]: Oops, missed that comment. I used @ApCoder123's suggested png

**Review by ApCoder123 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**